src/components/SelectionList/BaseSelectionListItemRenderer.tsx

```diff
@@ -72,11 +73,15 @@ function BaseSelectionListItemRenderer<TItem extends ListItem>({
                 isMultilineSupported={isMultilineSupported}
                 isAlternateTextMultilineSupported={isAlternateTextMultilineSupported}
                 alternateTextNumberOfLines={alternateTextNumberOfLines}
-                onFocus={() => {
+                onFocus={(event) => {
```

Comment:
```suggestion
                onFocus={(event: NativeSyntheticEvent<ExtendedTargetedEvent>) => {
```

And then we can replace `event?.nativeEvent` with `event.nativeEvent`

Reply:
Thanks! Working on it. I’ll update you later.

Reply:
I took `NativeSyntheticEvent` and `TargetedEvent` from [React Native](https://github.com/facebook/react-native/blob/f402ed17fa6d75aea24e2ad99a8b8d8ad20840e3/packages/react-native/Libraries/Components/Pressable/Pressable.d.ts#L77-L84), which are defined for platforms we don't use.

<img width="520" alt="Screenshot 2024-12-05 at 8 43 08 PM" src="https://github.com/user-attachments/assets/189b6dd0-efd2-4e5d-911f-bcb6c8210744">

<p></p>

However, the event is defined as `any` in [React Native Web](https://github.com/necolas/react-native-web/blob/a5ba27c6226aa182979a9cff8cc23c0f5caa4d88/packages/react-native-web/src/exports/View/types.js#L143), the actual platform where the code runs.

<img width="520" alt="Screenshot 2024-12-05 at 5 24 48 PM" src="https://github.com/user-attachments/assets/00871f99-29ac-4660-8697-3b92cd45fdf2">

Reply:
I think NativeSyntheticEvent is fine, as long as it doesn't produce any errors during runtime and during TS check. We use it across app already if I'm not mistaken